File: baekjoon_all/28000+/boj_28927.cpp
// Solve 2023-12-23

#include <iostream>
using namespace std;

int main() {
    int t1, e1, f1;
    cin >> t1 >> e1 >> f1;

    int t2, e2, f2;
    cin >> t2 >> e2 >> f2;

    int x1 = t1 * 3 + e1 * 20 + f1 * 120;
    int x2 = t2 * 3 + e2 * 20 + f2 * 120;

    if (x1 > x2) cout << "Max\n";
    else if (x1 < x2) cout << "Mel\n";
    else cout << "Draw\n";

    return 0;
}

File: baekjoon_all/28000+/boj_28927.py
# Solve 2023-12-23

a = [3, 20, 120]
t1 = input().split()
t2 = input().split()

t1 = sum([int(t1[i]) * a[i] for i in range(3)])
t2 = sum([int(t2[i]) * a[i] for i in range(3)])

if t1 > t2:
    print("Max")
elif t1 < t2:
    print("Mel")
else:
    print("Draw")
